tests/storage/test_event_federation.py

```diff
@@ -871,7 +869,44 @@ def test_get_backfill_points_in_room_attempted_event_retry_after_backoff_duratio
             self.store.get_backfill_points_in_room(room_id, depth_map["A"], limit=100)
         )
         backfill_event_ids = [backfill_point[0] for backfill_point in backfill_points]
-        self.assertListEqual(backfill_event_ids, ["b3", "b2", "b1"])
+        self.assertEqual(backfill_event_ids, ["b3", "b2", "b1"])
+
+    def test_repeated_get_backfill_points_in_room_succeed(self) -> None:
+        """
+        A test that reproduces #13929 (Postgres only).
+
+        We should be able to backoff repeatedly, even if the backoff formula would tell
+        us to wait for more seconds than can be expressed in a 32 bit signed int.
+        """
```

Comment:
This test name and description has caught me up a couple times now.

It's ok but I feel like it's not accurate. It's more:


```suggestion
    def test_get_backfill_points_in_room_works_after_many_failed_pull_attempts_that_could_naively_overflow(self) -> None:
        """
        A test that reproduces #13929 (Postgres only).

        Test to make sure we can still get backfill points after many failed pull
        attempts that cause us to backoff to the limit. Even if the backoff formula
        would tell us to wait for more seconds than can be expressed in a 32 bit
        signed int.
        """
```

Reply:
Thanks, I struggled to explain this and your words are much more meaningful!